### ISSUE TITLE
Could not read config file

### ISSUE THREAD

**Issue by 1nerdyguy** [markdomansky/WebJEA#45]
Fresh install, getting a "could not read config file" when attempting to load. 

Verified the web.config is pointing to C:\Scripts\config.json for the Webjea.my.mysettings variable. 
Verified the user for the App pool has read rights to that, as needed. 

In webjea.log, I find it referencing the following:
2020-12-29 14:07:21.7165|Debug|70||Cxxxxx\user|Looking for web.config/applicationSettings/WebJEA.My.MySettings/settings=configfile

2020-12-29 14:07:21.7165|Trace|71||Cxxxxx\user|GetFileContent: C:\prj\WebJEA CE\WebJEATest\configlocal.json

I went ahead and made that path, copied over the config.json and renamed it configlocal.json, but t hen I get a "Failed to map the path 'aspnet_client/system_web/4_0_30319/WebUIValidataion.js'

Any hints? I'm sure I screwed something up

**Comment by 1nerdyguy** [comment]
As an aside, I have now tried on three seperate VM's, verifying I didn't fack up the first one. Same exact error.

**Comment by markdomansky** [comment]
I'm looking into this.

**Comment by 1nerdyguy** [comment]
Just tried this again, and here is what i'm seeing in the logs still:
`2021-01-08 16:54:48.9618|Debug|570||Cxxxxxxxxxxxxx\hendersonb|Looking for web.config/applicationSettings/WebJEA.My.MySettings/settings=configfile
2021-01-08 16:54:48.9618|Trace|571||Cxxxxxxxxxxxxxx\hendersonb|GetFileContent: C:\prj\WebJEA CE\WebJEATest\configlocal.json`

This time, I tried just providing it a straight up username/password for the app pool account instead of a gmsa....didn't seem to make a difference though. Still referencing that non-existant path. 

Confirmed again my web.config is pointing to c:\scripts\config.json , and confirmed that the account has full control of the folder/files just to make sure.
